akka-testkit/src/main/scala/akka/testkit/TestFSMRef.scala

```diff
@@ -99,4 +99,14 @@ object TestFSMRef {
     val impl = system.asInstanceOf[ActorSystemImpl] //TODO ticket #1559
     new TestFSMRef(impl, Props(factory), impl.guardian.asInstanceOf[InternalActorRef], name)
   }
+
+  def apply[S, D, T <: Actor: ClassTag](factory: ⇒ T, supervisor: ActorRef, name: String)(implicit ev: T <:< FSM[S, D], system: ActorSystem): TestFSMRef[S, D, T] = {
+    val impl = system.asInstanceOf[ActorSystemImpl] //TODO ticket #1559
```

Comment:
Sure about the TODO issue nr? https://github.com/akka/akka/pull/1559 seems resolved